README.md

```diff
@@ -21,6 +21,14 @@ You can run the app in development mode. Open [http://localhost:3000](http://loc
 
 `yarn start`
 
+> __Note:__ If you are using Node v17+ you may get `error:03000086:digital envelope routines::initialization error` error when running `yarn start`. If so please follow the instructions on https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported to resolve. 
+
 You can run the test runner in the interactive watch mode.
 
 `yarn test`
+
+### Running against a local instance of the registry-api
```

Comment:
I'm sure there is a *better* way to do this, something like pointing back to variable in server.js for example, but I'm not confident enough to try that right now and this works!

Please feel free to remove or update this section if there is a more obvious approach.